app/preprints/index/template.hbs

```diff
@@ -96,7 +96,7 @@
                 </p>
                 <div local-class='preprint-logos-container'>
                     {{#each this.model.brandedProviders as |provider| }}
-                        {{#if (not-eq provider.id 'livedata') }}
+                        {{#if (and provider.advertiseOnDiscoverPage (not-eq provider.id 'livedata'))}}
```

Comment:
Instead of filtering on the front end, it would make more sense to filter on the API request. Also, we shouldn't need the livedata exception anymore, as long as we make sure in the release playbook that we uncheck the advertise flag for livedata as well as lawrxiv. This whole if statement should be removable.